pom.xml

```diff
@@ -289,6 +289,48 @@ under the License.
 				<version>1.8.2</version>
 			</dependency>
 
+			<!-- For dependency convergence -->
+			<dependency>
+				<groupId>org.hamcrest</groupId>
+				<artifactId>hamcrest-core</artifactId>
+				<version>${hamcrest.version}</version>
+			</dependency>
+
+			<!-- For dependency convergence -->
+			<dependency>
+				<groupId>org.objenesis</groupId>
+				<artifactId>objenesis</artifactId>
+				<version>2.2</version>
```

Comment:
is there an existing dependency that pulls in 2.2? (I only found 2.1 usages)

Reply:
Indeed it is strange, I don't see 2.2 in `dependency:tree`, but without forcing 2.2 version there is this convergence error:
```
Dependency convergence error for org.objenesis:objenesis:2.2 paths to dependency are:
+-org.apache.flink:flink-parent:1.4-SNAPSHOT
  +-org.powermock:powermock-module-junit4:1.6.5
    +-org.powermock:powermock-module-junit4-common:1.6.5
      +-org.powermock:powermock-reflect:1.6.5
        +-org.objenesis:objenesis:2.2
and
+-org.apache.flink:flink-parent:1.4-SNAPSHOT
  +-org.powermock:powermock-api-mockito:1.6.5
    +-org.mockito:mockito-core:1.10.19
      +-org.objenesis:objenesis:2.1
```

Reply:
If possible i would downgrade this to 2.1, as this is the version used by the runtime dependencies (in particular kryo).

Reply:
Make sense. Downgraded to 2.1.